tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -84,12 +84,21 @@ public SecuritySettings(IConfigurationSource? source, IConfigurationTelemetry te
 
             ApiSecurityEnabled = config.WithKeys(ConfigurationKeys.AppSec.ApiExperimentalSecurityEnabled)
                                        .AsBool(false);
+            var defaultValueUseLegacyEncoder = false;
+#if NETFRAMEWORK
+            defaultValueUseLegacyEncoder = true;
+#endif
```

Comment:
I'm open to be convinced otherwise, I think we should set this to false for everyone, gather some beta testers, before setting it to true.

Reply:
changed in [fd8ae19](https://github.com/DataDog/dd-trace-dotnet/pull/4884/commits/fd8ae197a13170604645043af6b22181a7d3039d) to false by default

Reply:
Actually, `false` means the new encoder is the active one by default, which is the opposite of what I meant.

Default should be true, or maybe we should change the setting mean to "use new encoder" and keep the default at false.

Reply:
Fixed in 5a6f86a5b